hlapipeline/alignimages.py

```diff
@@ -294,11 +295,14 @@ def perform_align(input_list, archive=False, clobber=False, update_hdr_wcs=False
     # 7: Write new fit solution to input image headers
     print("-------------------- STEP 7: Update image headers with new WCS information --------------------")
     if update_hdr_wcs:
-        update_image_wcs_info(imglist, processList)
+        updated_image_list,headerlet_list = update_image_wcs_info(imglist, processList)
         print("\nSUCCESS")
     else:
+        updated_image_list = []
+        headerlet_list = []
         print("\n STEP SKIPPED")
-    return (0)
+    output_dictionary = generate_final_output_dict(imglist,updated_image_list,headerlet_list,best_fit_rms)
+    return (output_dictionary)
```

Comment:
The return to the calling routine is the filteredTable.  At this stage the only item missing from the table is the name of the headerlet file for each image in the dataset.

Reply:
Issue resolved by commit [ba12b2b](https://github.com/spacetelescope/hlapipeline/pull/56/commits/ba12b2b9efeb2bb783734e9e3f8cff62eaa07280)